src/components/About.tsx

```diff
@@ -1,7 +1,35 @@
 import React from "react";
+import Image from "next/image";
+import swLogo from "@/public/swNameOnly.svg";
 
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="flex w-full flex-row gap-x-[10%]">
+      <div className="flex w-auto flex-col gap-y-4 pl-[12%] pt-[9%]">
+        <p className="ml-2 bg-gradient-to-b from-[#DDA82A] via-sw-white to-[#905803] bg-clip-text font-bold text-transparent sm:text-4xl md:text-5xl lg:text-6xl">
```

Comment:
use sw-gold-100 and sw-gold-200 instead of hex codes

Reply:
ok i changed them